window/src/os/macos/window.rs

```diff
@@ -397,10 +398,7 @@ impl Window {
 
             window.setReleasedWhenClosed_(NO);
             window.setOpaque_(NO);
```

Comment:
I think the artifacts in the other issue came about because this opacity boolean is NO and shadow is YES.  Just enabling the shadows when the alpha level is opaque may not prevent artifacts from showing up (not sure).

Reply:
Ah, so if we change opaque correctly here we can have avoid that issue even with drop shadow on?

Reply:
We have to have opaque = NO for alpha < 1.0 to work.  But when the window is not opaque, drop shadows result in artifacts.  I don't know if the artifacts are caused by the alpha not being 1.0 or if it is just because opaque=NO (or both). 

I think to be safest, we should set both opaque and drop shadow flags to the appropriate values based on the configured opacity level.

Reply:
Ah, I only realized now that my understanding of "opaque" is the opposite of what it means (hence the direction I set in the last commit is incorrect). 😬